edx_sga/sga.py

```diff
@@ -57,69 +57,69 @@ class StaffGradedAssignmentXBlock(XBlock):
     icon_class = 'problem'
 
     display_name = String(
-        default='Staff Graded Assignment', scope=Scope.settings,
-        help="This name appears in the horizontal navigation at the top of "
-             "the page."
+        default= "Staff Graded Assignment", scope=Scope.settings,
```

Comment:
To be strict, the problem here is the extra space after the equals, since these are keyword arguments to String().